src/useChunk.js

```diff
@@ -1,26 +1,20 @@
 // @ts-check
-import { useContext, useEffect, useState } from "react";
+import { useContext } from "react";
+import useSWR from 'swr';
 
 import { api } from "./api";
 import { EditmodeContext } from "./EditmodeContext";
 import { renderChunk } from "./utils/renderChunk.jsx";
 import { computeContentKey } from "./utils/computeContentKey";
-import { Platform, AsyncStorage } from 'react-native';
 
 export function useChunk(defaultContent, { identifier, type }) {
-  const { projectId, chunkFallback } = useContext(EditmodeContext);
-  const [[error, chunk], setResponse] = useState([undefined, undefined]);
+  const { projectId, defaultChunks } = useContext(EditmodeContext);
   const contentKey = defaultContent ? computeContentKey(defaultContent) : null;
   const url = identifier
     ? `chunks/${identifier}`
     : `chunks/${contentKey}?project_id=${projectId}`;
 
-  useEffect(() => {
-    api
-      .get(url)
-      .then((res) => setResponse([null, res.data]))
-      .catch((error) => setResponse([error, null]));
-  }, [url]);
+  const { data: chunk, error } = useSWR(url, api.get(url).then((res) => res));
```

Comment:
You probably won't need `.then(res => res)` here, if it's working already.

Reply:
Will the value for `fallbackChunk` (we can rename that variable to `defaultChunk`) be used for `initialData`?